app/controllers/cim_instance_controller.rb

```diff
@@ -92,6 +92,8 @@ def process_button
       end
     end
 
+    return if performed? # return if another method has already redirected or rendered
```

Comment:
@h-kataria : I'd say that the comment is not needed here (and on the other places)